src/components/gameScreen/gameScreen.component.js

```diff
@@ -57,18 +61,18 @@ export const GameScreenComponent = observer(({
           <View style={styles.customButtonsContainer}>
             <CustomButton
               img={questionIcon}
-              color={store.currentCategory.textColor}
+              color={categoriesStore.currentCategory.textColor}
               onTouch={onOpenHelpDialog}
               setButtonPressed={false}
             />
             <View style={styles.timer}>
-              {(store.chosenMode === 'Hard') ?
+              {(dialogsStore.chosenMode === 'Hard') ?
```

Comment:
in the moments like this try to move conditional logic in variables. Create a variable in the component called isHardMode = dialogsStore.chosenMode === 'Hard' and use it here